json/pGTW.json

```diff
@@ -2004,15 +2036,15 @@
       "rulings": [
         {
           "date": "2008-08-01",
-          "text": "Each of the triggered abilities has an \"intervening 'if' clause.\" That means (1) the ability won't trigger at all unless you control two or more lands of the appropriate land type when the Hedge-Mage enters the battlefield, and (2) the ability will do nothing if you don't control two or more lands of the appropriate land type by the time it resolves."
+          "text": "Each of the triggered abilities has an \"intervening ‘if' clause.\" That means (1) the ability won't trigger at all unless you control two or more lands of the appropriate land type when the Hedge-Mage enters the battlefield, and (2) the ability will do nothing if you don't control two or more lands of the appropriate land type by the time it resolves."
```

Comment:
Here, a `'` is changed into a `‘`. This seems like the opposite of the intended change.